app/src/main/java/com/lucascabral/marvelsuperheroes/presenter/MarvelApp.kt

```diff
@@ -0,0 +1,19 @@
+package com.lucascabral.marvelsuperheroes.presenter
```

Comment:
Algum motivo específico pra esse pacote se chamar `presenter`?

Reply:
Na arquitetura clean usamos presenter para a camada de apresentação (ui/adapter/), no caso penso em usar as duas arquiteturas no futuro clean e mvvm - se for interessante.